public/css/application.css

```diff
@@ -46,3 +46,13 @@
 	margin-top: -3rem;
 	width: 100%;
 }
+
+.button--red{
+	color: white;
+	background-color:red;
+}
+
+.button--red-hover:hover{
```

Comment:
So I think even in BEM, this hover class is self-documenting— since the pseudoclass is called `:hover`, and describes only hover-y use cases, you don't need the further specifier for `.button--red`. 

To rephrase: we can assume that the specified `.button--red` class has behavior particular to it— behavior which includes both its attributes (i.e. `background: red`) and its pseudoclasses. That way we can avoid overly bloated classes on HTML.